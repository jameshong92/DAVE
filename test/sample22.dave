// test table binding1
int main() {
	string s[] = ["a", "b", "c", "d", "e"];
	fld t1 = fld(s, "eng");
    fld t2 = fld([1, 2, 3, 4, 5], "num");
    fld[] t = [t1, t2];
    tbl test = tbl(t);
    print(test);
    string n[] = ["z", "y", "x", "w", "v"];
    fld t3 = fld(n, "rev");
    test = append(test, t3);
    print(test);
}