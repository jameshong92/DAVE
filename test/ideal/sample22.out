eng	num	
a	1	
b	2	
c	3	
d	4	
e	5	

eng	num	rev	
a	1	z	
b	2	y	
c	3	x	
d	4	w	
e	5	v	

